lib/Backend/GlobOpt.cpp

```diff
@@ -6417,6 +6420,38 @@ GlobOpt::GetConstantVar(IR::Opnd *opnd, Value *val)
     return nullptr;
 }
 
+Js::Var
+GlobOpt::GetConstantVar(IR::Opnd *opnd)
```

Comment:
>GetConstantVar [](start = 9, length = 14)

nit: can we refactor the logic of this function into the GetConstantVar above. and pass nullptr from places where you call this function?